app/views/donations/show.html.erb

```diff
@@ -75,7 +75,10 @@
               enabled: !@donation.has_inactive_item?,
               size: "md" } %>
             <%= new_button_to new_distribution_path(donation_id: @donation.id, storage_location_id: @donation.storage_location_id), { text: "Start a new Distribution" } %>
-            <%= delete_button_to donation_path(@donation), { size: "md", confirm: "Are you sure you want to permanently remove this donation?" } if current_user.has_role?(Role::ORG_ADMIN, current_organization) %>
+            <%= delete_button_to donation_path(@donation), {
+              size: "md",
+              enabled: !@donation.has_inactive_item?,
+              confirm: "Are you sure you want to permanently remove this donation?" } if current_user.has_role?(Role::ORG_ADMIN, current_organization) %>
```

Comment:
This is a pretty hefty statement now - can we break the `if` into the block format instead of it being at the end of it?

Reply:
@dorner Thanks for the suggestion. The change has been made.